kore/src/Control/Monad/Trans/Accum/Stricter.hs

```diff
@@ -0,0 +1,221 @@
+{-# LANGUAGE NoStrict #-}
+{-# LANGUAGE NoStrictData #-}
+{-# LANGUAGE ImplicitPrelude #-}
+
+-----------------------------------------------------------------------------
+
+-----------------------------------------------------------------------------
+
+{- |
+ Module      :  Control.Monad.Trans.Accum.Stricter
+ Copyright   :  (c) Nickolay Kudasov 2016
```

Comment:
@treeowl was this module extracted from some package? Why not use the package directly? Or is this your code and you forgot to modify the copyright?

@ehildenb is this an issue?

Reply:
It's modified from the code in `transformers`. If it proves useful, we can do whatever's appropriate with the copyright notice.